Docs/Architecture/Simulation.md

```diff
@@ -2,35 +2,53 @@
+The sim graphical user interface (GUI) looks like this and can be launched through the WPILib command palette.
+
+<img src="../../Assets/SimGUI.webp" alt="Screenshot of the sim GUI" width="600"/>
+
+From here, you can view some interesting bits of information about your simulated robot, such as its state, timing, joysticks, etc.
+
+Then we have WPILib’s simulation physics classes.
+These are built around a state-space model (a set of matrix equations that describe how a system changes over time) of a class of mechanism and exist for many common mechanisms to predict how they will behave.
+Like other simulated systems, the mechanism’s state is updated periodically with simulated inputs.
+For example, we can simulate the position of an elevator after calculating and applying a certain voltage to its motor.
+
+Sometimes these models will be very accurate.
+Sometimes they will make a lot of untrue assumptions about the mechanism.
+Both are useful, since we will likely have to retune the mechanism one way or another once we have real hardware.
+Deciding how much effort to put into a simulation model is a key part of the intuition you will build as you code more robots.
 We tend to use one or two of these per robot for major mechanisms.
-If you want to learn more about this sort of modelling, look at the [control theory for FRC book](https://file.tavsys.net/control/controls-engineering-in-frc.pdf).
+If you want to learn more about this sort of modeling, look at the [control theory for FRC book](https://file.tavsys.net/control/controls-engineering-in-frc.pdf).
 Note that the topics covered by this book are largely far more advanced than high schoolers have the math background for, and it is not required reading.
 
 Running a model is useful, but even more useful is being able to visualize the output of the model.
-[AdvantageScope](https://github.com/Mechanical-Advantage/AdvantageScope) is a tool developed by team 6328 to visualize data from live robots, simulated robots, and robot logs.
+There are several dashboards you can use to do this, but we mainly use AdvantageScope.
+[AdvantageScope](https://github.com/Mechanical-Advantage/AdvantageScope) is a tool developed by Team 6328 to visualize data from live robots, simulated robots, and robot logs.
 It provides a variety of formats to visualize data from line graphs to 3d field models to tables and more.
-It is closely integrated with AdvantageKit but does not require it.
 
-The final major tool is being able to swap between simulated inputs and outputs (io) and real io easily and correctly.
-This is the real advantage of the AdvantageKit Subsystem -> IO -> IOImplementation structure.
-By having a simulation and real IOImplementation for each mechanism we can easily test our code in sim and have minimal work to move it to real hardware.
+<img src="../../Assets/AdvantageScopeScreenshot.png" alt="Screenshot of Advantagescope" width="600"/>
```

Comment:
its probably worth having a more in depth breakdown of common tools in ascope, but i dont think that fits here. maybe in the sim kitbot tutorial?

Reply:
yeah i can add that there